### PR TITLE
#166214079 Basic  Server Unit test 

### PR DESCRIPTION
**Pull request by Adekoreday**
[Delivers #166214079]

#### Description
-ensure the server is up and running with minimal error
#### Type of change
Test
#### How Has This Been Tested?
```
clone repository
create a .env file in the project root directory
add enviroment variable PORT=your port value
npm run test
```

#### Checklist:
N/A
#### JIRA
N/A
#### PIVOTAL STORY
https://www.pivotaltracker.com/story/show/166214079

### CODE REVIEW

File: test/index-test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+                });
+        });
+        it('its expected to return 404 on not found request', (done) => {
+chai.request(server)
+                .get('/adekorede')
+                .end((err, res) => {
+                    expect(res, 'must have a status 404 not found').to.have.status(404);
+                    done();
+                });
+        })
```
[the remark this answers is not in the document]
Comment (Adekoreday) [reply test/index-test.js]: fixed this

**Comment by JuwonAbiola** [comment]
In your PR, your pivotal story should have a link i.e [id](link) and it's Pivotal Tracker not JIRA
noted will look into it thanks

**Comment by allebd** [comment]
You will need to rebase your branch with develop
ok noted thanks

**Comment by allebd** [comment]
eslint was removed from package.json and kindly update your nyc to 14.1.1, it shows high severity vulnerability
done

**Review by Lundii — APPROVED** [review]
Two test passed

**Review by allebd — APPROVED** [review]
100% test coverage

**Review by JuwonAbiola — APPROVED** [review]
works fine two test passed

**Review by KvNGCzA — APPROVED** [review]

**Pull request merged**